subsys/net/ip/net_if.c

```diff
@@ -141,6 +141,13 @@ static bool net_if_tx(struct net_if *iface, struct net_pkt *pkt)
 	struct net_context *context;
 	int status;
 
+#if defined(CONFIG_NET_CONTEXT_TIMESTAMP)
+	/** Timestamp of the current network packet sent */
+	struct net_ptp_time start_timestamp;
+	u32_t curr_time = 0;
+	u8_t pkt_priority;
```

Comment:
I'm surprised to see "pkt_priority" to be wrapped in "CONFIG_NET_CONTEXT_TIMESTAMP".